hrrecruitment/tests/phpunit/CRM/HRRecruitment/BAO/HRVacancyTest.php

```diff
@@ -18,12 +18,6 @@ public function setUpHeadless() {
       ->apply();
   }
 
-  function setUp() {
-  }
-
-  function teardown() {
-  }
-
   protected static function _populateDB($perClass = FALSE, &$object = NULL) {
```

Comment:
Note: Please don't forget to remove the same method in CRM_HRRecruitment_BAO_HRSearchVacancyTest as well.